### ISSUE TITLE
ionic-select-searchable has no exported member 'SelectSearchable'

### ISSUE THREAD

**Issue by sudhirbitsgoa** [eakoriakin/ionic-select-searchable#1]

**Comment by eakoriakin** [comment]
Hi @sudhirbitsgoa, are you using lazy loading by any chance?
If you do, try importing SelectSearchable to each "lazy" page module.

```
import { LazyPage } from './lazy';
import { SelectSearchableModule } from 'ionic-select-searchable';

@NgModule({
	declarations: [
		LazyPage
	],
	imports: [
		IonicPageModule.forChild(LazyPage),
		SelectSearchableModule
	]
})
export class LazyPageModule { }
```

**Comment by eakoriakin** [comment]
@sudhirbitsgoa, try this:
‘import { SelectSearchable } from 'ionic-select-searchable';’

**Comment by eakoriakin** [comment]
@sudhirbitsgoa, I fixed it in a **1.0.9** version.

**Comment by sudhirbitsgoa** [comment]
Ok will check. Thanks
